scripts/rhv_tier_analyzer.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+import argparse
+import subprocess
+import sys
+
+PROVIDER = 'rhv_cfme_integration'
```

Comment:
**Required**

Make these CLI args, so that the script has some more flexibility/function to it - we can then run it against generic tier markers, and any provider we want.

Since PROVIDER is passed to `pytest` call, it can be passed on CLI as anything that matches for that use  - a provider_key or a tag.